packages/database/src/migrations/20210222124639-UpdateLaosEntitiesPushMetadata-modifies-data.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NEW_METADATA = {
+  dhis: {
+    push: false,
+    isDataRegional: true,
+  },
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- We want to drop the entity trigger because we just want to add push: false to Laos entities, no actual data is changed, this shouldn't trigger any change listener (eg: for pushing)
+    DROP TRIGGER IF EXISTS entity_trigger
+    ON entity;
+
+    -- Add push = false to Laos entities
+    UPDATE entity
+    SET metadata = '${JSON.stringify(NEW_METADATA)}'
+    WHERE country_code = 'LA';
+
+    -- Also change the metadata in data column if dhis_sync_log to reflect the same push = false
+    UPDATE dhis_sync_log
+    SET data = jsonb_set(data::jsonb, '{metadata}', '${JSON.stringify(NEW_METADATA)}')::text
+    WHERE record_type = 'entity'
+    AND record_id IN (SELECT id FROM entity WHERE country_code = 'LA');
```

Comment:
There are some existing sync_logs for Laos entities. I'm not sure if we can/should delete them...? So this is just to update the `data` column to also have the same `push` flag = false

Reply:
I think it's fine either way to delete or leave. They're kind of just clutter, but it's also a very cluttered table anyway! They probably represent the sync to the regional dhis2 server, which is valid, but if we deleted the Laos entities from there (which we may want to), then we should definitely delete both the sync logs and sync queue records. For now, up to you.